src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,33 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const checkComments = () => {
+    if (post.comments && post.comments.length > 0) {
+      return <CommentList comments={post.comments} />;
+    }
+
+    return <b data-cy="NoCommentsMessage">No comments yet</b>;
+  };
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={post.user} />
```

Comment:
find can return undefined and user will not exist, so better to add the check here
```suggestion
          {post.user && <UserInfo user={post.user} />}
```